### PR TITLE
Interaction with delete

### PR DESCRIPTION
**Pull request by Richard908**

### CODE REVIEW

**Comment by frozar** [comment]
Lors que j'appuie sur `Echappe`, je ne sors pas du mode SuppressionLine. A corriger.

File: src/userInformation/utils.ts [hunk 1]
```diff
@@ -14,3 +15,14 @@ export function displayAddLineMessage() {
     });
   }
 }
+
+export function displayRemoveLineMessage() {
+  if (getMode() === ModeEnum.removeLine) {
```
Comment (hugobaret) [inline src/userInformation/utils.ts:20]: isRemoveLineMode()

**Review by hugobaret — COMMENTED** [review]
Change the condition getMode() == ModeEnum.RemoveLineMode. cf comments.

File: src/userInformation/utils.ts (hunk 1 above)
Comment (Richard908) [reply src/userInformation/utils.ts:20]: Done.
isRemoveLineMode() => is**In**RemoveLineMode()

**Review by hugobaret — APPROVED** [review]

**Pull request merged**